Remotion/Web/Core/ExecutionEngine/WxeStepList.cs

```diff
@@ -174,9 +174,9 @@ private void InitializeSteps ()
         }
         else if (member is Type)
         {
-          Type subtype = member as Type;
+          Type? subtype = member as Type;
```

Comment:
That's just a stupid cast (check the IF in line 175)
```suggestion
          Type? subtype = (Type) member;
```